lib/authlogic/acts_as_authentic/session_maintenance.rb

```diff
@@ -66,38 +66,38 @@ def self.included(klass)
             before_save :maintain_sessions, :if => :update_sessions?
           end
         end
-        
+
         # Save the record and skip session maintenance all together.
-        def save_without_session_maintenance(*args)
+        def save_without_session_maintenance(validate=true)
```

Comment:
@SamuelMartini I think `*args` is actually a good option here since it's more-or-less a mixin method. I think the idea is that this method should act like `save`, and `*args` is a good conventional way to do that (ie you could replace `save(*args)` with `super` for the same result). 

Is this method being called directly from your app, and passing in `false` instead of an options hash?